kernel_tuner/cache/json.py

```diff
@@ -1,14 +1,18 @@
-from typing import TypedDict, NotRequired, Any
+from __future__ import annotations
+from typing import TypedDict, Any
 
 
 _non_identifier_keys = TypedDict("CacheEntryJSON", {
-    "GFLOP/s": NotRequired[float]
```

Comment:
Je had in een eerdere commit de `CacheEntryJSON` class hernoemd naar `CacheLineJSON`, maar hier staat nog wel `"CacheEntryJSON"`. Is dat bewust? Heeft niks te maken met deze merge request overigens.

Reply:
Oh, dat kan ik nog even veranderen als het handig is, maar we zouden het ook nog kunnen verplaatsen naar een toekomstige code cleanup